recipes/libheif/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class Libheif(ConanFile):
+    name = "libheif"
+    description = "libheif is an HEIF and AVIF file format decoder and encoder."
+    topics = ("conan", "libheif", "heif", "codec", "video")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/strukturag/libheif"
+    license = ("LGPL-3.0-only", "GPL-3.0-or-later", "MIT")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False],
+               "with_x265": [True, False]}
+    default_options = {"shared": False, "fPIC": True,
+                       "with_x265": False}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = '{}-{}'.format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def requirements(self):
+        self.requires("libde265/1.0.8")
+        if self.options.with_x265:
+            self.requires("libx265/3.2.1")
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        for f in ["X265", "Libde265", "LibAOM"]:
+            os.unlink(os.path.join(self._source_subfolder, "cmake", "modules", "Find"+f+".cmake"))
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
+        self._cmake.definitions["WITH_EXAMPLES"] = False
+        self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "heif"
+        self.cpp_info.names["cmake_find_package_multi"] = "heif"
+        self.cpp_info.names["pkg_config"] = "heif"
```

Comment:
I think it's [`libheif`](https://github.com/strukturag/libheif/blob/5b3c8dd984b110d2921378609ce27e98afe9bda5/Makefile.am#L15)
```suggestion
        self.cpp_info.names["pkg_config"] = "libheif"
```

Reply:
Yes, confirmed by `libheif.pc.in`